mne_icalabel/features/psd.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from mne.io.pick import _picks_to_idx
+from mne.time_frequency.psd import psd_multitaper
+from mne.utils import _check_option
+from mne.viz.ica import _prepare_data_ica_properties
+
+
+def get_psd(
```

Comment:
```suggestion
def get_psds(
```

I would rename this one with an extra `s` to match the naming schemes of the topomap feature.